node-js-basics-101/node_modules/cookie/index.js

```diff
@@ -0,0 +1,195 @@
+/*!
+ * cookie
+ * Copyright(c) 2012-2014 Roman Shtylman
+ * Copyright(c) 2015 Douglas Christopher Wilson
+ * MIT Licensed
+ */
+
+'use strict';
+
+/**
+ * Module exports.
+ * @public
+ */
+
+exports.parse = parse;
+exports.serialize = serialize;
```

Comment:
__[jshint]__
* 'exports' is not defined.

<!-- Comment by [SideCI](https://sideci.com) -->